teuthology/repo_utils.py

```diff
@@ -218,12 +269,16 @@ def reset_repo(repo_url, dest_path, branch):
                       GitError for other errors
     """
     validate_branch(branch)
-    log.info('Resetting repo at %s to branch %s', dest_path, branch)
+    if '/' in branch:
+        reset_branch = lsstrip(remote_ref_from_ref(branch), 'refs/remotes/')
+    else:
+        reset_branch = 'origin/' + branch
```

Comment:
Why change `'origin/%s' % branch` to `'origin/' + branch` here?

Reply:
It was changed with no special intend.